docs/csharp/language-reference/builtin-types/floating-point-numeric-types.md

```diff
@@ -0,0 +1,99 @@
+ms.date: 06/30/2019
+f1_keywords: 
+  - "float"
+  - "float_CSharpKeyword"
+  - "double"
+  - "double_CSharpKeyword"
+  - "decimal_CSharpKeyword"
+  - "decimal"
+helpviewer_keywords: 
+  - "floating-point numbers [C#]"
+  - "ranges of floating-point types [C#]"
+  - "types [C#], floating-point types"
+  - "float keyword [C#]"
+  - "floating-point numbers [C#], float keyword"
+  - "double data type [C#]"
+  - "decimal keyword [C#]"
+---
+# Floating-point types
+
+The **floating-point types** are a subset of the **simple types** and can be initialized with [*literals*](#floating-point-literals). All floating-point types are also value types. All floating point numeric types support [arithmetic](../operators/arithmetic-operators.md) [comparison, and equality](../operators/equality-operators.md) operators.
+
+The following table shows the precision and approximate ranges for the floating-point types.  
+  
+|Type|Approximate range|Precision|  
+|----------|-----------------------|---------------|  
+|`float`|±1.5 x 10<sup>−45</sup> to ±3.4 x 10<sup>38</sup>|~6-9 digits|  
+|`double`|±5.0 × 10<sup>−324</sup> to ±1.7 × 10<sup>308</sup>|~15-17 digits|  
+|`decimal`|±1.0 x 10<sup>-28</sup> to ±7.9228 x 10<sup>28</sup>|28-29 digits|  
+
+The default value for all floating-point types is `0`. Each of the floating-point types has constants named `MinValue` and `MaxValue` for the minimum and maximum value for that type. The `float` and `double` types have additional constants for `PositiveInfinity`, `NegativeInfinity`, and `NaN` (for "Not a Number"). The `decimal` type includes constants for `Zero`, `One`, and `MinusOne`.
+
+The `decimal` type has more precision and a smaller range than both `float` and `double`, which makes it appropriate for financial and monetary calculations.
+
+You can mix numeric integral types and floating-point types in an expression. In this case, the integral types are converted to floating-point types. The evaluation of the expression is performed according to the following rules:
+
+- If one of the floating-point types is `double`, the expression evaluates to `double`, or to [bool](../keywords/bool.md) in relational comparisons or comparisons for equality.
+- If there is no `double` type in the expression, the expression evaluates to `float`, or to [bool](../keywords/bool.md) in relational comparisons or comparisons for equality.
+
+A floating-point expression can contain the following sets of values:
+
+- Positive and negative zero
+- Positive and negative infinity
+- Not-a-Number value (NaN)
+- The finite set of nonzero values
+
+For more information about these values, see IEEE Standard for Binary Floating-Point Arithmetic, available on the [IEEE](https://www.ieee.org) website.
+
+## floating point literals
+
+By default, a real numeric literal on the right side of the assignment operator is treated as `double`. You can use suffixes to convert a literal to a specific type. 
```

Comment:
Two remarks:
- **real** integral is not defined (section is called "Floating-point literal")
- is it worth to mention that any integral literal is a valid floating-point literal?
```suggestion
By default, a floating-point numeric literal on the right side of the assignment operator is treated as `double`. You can use suffixes to convert a literal to a specific type. 
```